docs/en_US/TrainingService/AMLMode.rst

```diff
@@ -125,3 +125,8 @@ Run the following commands to start the example experiment:
    nnictl create --config config_aml.yml
 
 Replace ``${NNI_VERSION}`` with a released version name or branch name, e.g., ``v2.0``.
+
+Monitor your code in the cloud by using the studio
```

Comment:
"cloud" and "studio" are the words used in AML?

Reply:
Yes.  It comes from [here](Words used in AML docs.  It comes from [here](https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-1st-experiment-hello-world#monitor)

Reply:
got it, thanks